src/common_i2c.cpp

```diff
@@ -290,6 +311,33 @@ int i2cDev::i2c_transfer(uint8_t tx_len, uint8_t* tx_data, uint8_t rx_len,
     return res;
 }
 
+bool i2cDev::isSpamingToLog(int res, std::stringstream& ss)
```

Comment:
I dislike that logging functionality moved from the calling side (by passing `ss`) and prefer this method to keep silence or log the self owned warnings only.

Reply:
fixed